app/templates/inbox/meetings/new.hbs

```diff
@@ -0,0 +1,70 @@
+{{!-- TODO: Uncomment this once the app uses `{{page-title}}` on all relevant pages --}}
+{{!-- {{page-title (t "inbox.meetings.new.title")}} --}}
+
+<AuModal
+  @modalTitle={{t "inbox.meetings.new.title"}}
+  @modalOpen={{true}}
+  @closeModal={{this.cancelMeetingCreation}}
+  as |Modal|
+>
+  <Modal.Body>
+    <form class="au-o-flow" id="create-meeting-form" {{on "submit" (perform this.saveMeetingTask)}}>
+      <div>
+        {{#let (unique-id) this.meeting.errors.bestuursorgaan as |id errors|}}
+          <AuLabel
+            @error={{if errors true false}}
+            for={{id}}
+          >
+            {{t "inbox.meetings.new.meeting.administrativeBody"}} <AuPill>Verplicht</AuPill>
```

Comment:
Forgot to translate the "Verplicht" part. But I was thinking that a small template only component would make more sense? I'm guessing there will be other places where it's needed.

Reply:
Created a simple `<RequiredField>` template only component for this. Let me know if it needs to be named differently :smile:.